visualizer/parseTree.js

```diff
@@ -717,8 +718,10 @@
   // -------
 
   var parseTree = ohmEditor.parseTree = new CheckedEmitter();
-  parseTree.refresh = refreshParseTree;
-
+  parseTree.refresh = function(rootTrace, clearZoomState) {
+    clearMarks();
```

Comment:
I think maybe we should just be clearing the marks inside `refreshParseTree`. What do you think?